chef/automate
# Event Feed project filtering for the GetFeed request

### :nut_and_bolt: Description: What code changed, and why?

Adding project filtering for the GetFeed endpoint event-feed-service. This is one of the three endpoints that need project filtering. Project filtering is only used for chef-server events which are not added yet. 

### :chains: Related Resources

https://github.com/chef/automate/issues/444

### :athletic_shoe: How to Build and Test the Change
This might be hard to test. 
1. `build event-feed-service && start_all_services`
1. Add a project named "project9"
1. Add a project named "project3"
1. Add a chef-server event tagged with "project9" run `event_feed_add_event` 
1. Create a profile event by adding a profile from the "Available" section here https://a2-dev.test/compliance/compliance-profiles
1. Request the feed for project3 only. Ensure that the chef-server event is not returned, but the profile event is returned
1. Request the feed for projects9 only. Ensure both events are returned. 

### :white_check_mark: Checklist
